book/src/dev/release-criteria.md

```diff
@@ -0,0 +1,113 @@
+# Go/No-Go Release Criteria
+
+## Alpha Release
+
+In the first alpha release, we want Zebra to participate in the network and replicate the Zcash chain state. We also want to validate proof of work and the transaction merkle tree, and serve blocks to peers.
+
+### System Requirements
+
+Our CI tests that Zebra builds, passes its tests, and syncs to tip on the following configurations:
+* Ubuntu Linux ...
+
+Our CI tests that Zebra builds and passes its tests on the following configurations:
+* recent macOS ...
+* recent Windows ...
+
+### Build Requirements
+
+Our CI builds Zebra with:
+* Rust stable
+* recent llvm
+* recent clang
+* recent libclang
+
+### Supported Platforms
+
+While Zebra is still in alpha, we don't guarantee support for any particular platforms.
+
+But you'll probably get the best results with a recent Ubuntu Linux, or the other platforms that our CI runs on.
+
+### Go/No-Go Criteria
+
+- `zebrad` Functionality
+    - [ ] `zebrad` can sync to mainnet tip
```

Comment:
how do these check boxes render in `mdbook`? I am worried this might be a github specific feature that wont translate to our book.

Reply:
let's use the emoji instead when we do the first update to the status of each item.